index.js

```diff
@@ -103,13 +103,23 @@ module.exports = async function(options) {
     // are out there and what icons they need without
     // actually instantiating them
     self.modulesToBeInstantiated = modulesToBeInstantiated;
+    self.eventAliases = {};
+    self.alias('modulesReady', 'modulesRegistered');
+    self.alias('afterInit', 'ready');
 
     defineModules();
 
     await instantiateModules();
     lintModules();
-    await self.emit('modulesReady');
-    await self.emit('afterInit');
+    await self.emit('modulesRegistered'); // formerly modulesReady
+    await self.apos.migration.migrate(); // emits before and after events, inside the lock
```

Comment:
I don't see an after event in the migrate method.

Reply:
Probably because that one is not new, so it's not in the diff. It's there.

Reply:
true.